tests/misc/test_cross_pool_migration.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+from lib.common import wait_for, wait_for_not
+
+pytestmark = pytest.mark.default_vm('mini-linux-x86_64-bios')
+
+def test_cross_pool_migration(host, hostB1, imported_vm):
+    assert host.pool.uuid != hostB1.pool.uuid
+
+    vm = imported_vm.clone()
```

Comment:
Why are you using a clone now?

Reply:
So that we can leave the original VM on the first pool, without having to move it back. Do you want me to add a comment?

Reply:
Note: this is particularly useful when used with `--vm={uuid of an existing VM}`

Reply:
No need, I was just curious ^^